component styles/Avatar.css

```diff
@@ -0,0 +1,52 @@
+.avatar-container{
+    display: flex;
+    width: 50%;
```

Comment:
Giving width like this will not give a smooth resizing approach. As you approach your breakpoint of 560px, the screen takes up the entire space. If you want to keep the content in middle all the time do the following
```
width: 90vw:
margin: 0 auto;
max-width: 700px; // you can choose any value instead of 700 px
```

what this does is, it keeps width at 90vw always and keeps it centered because of `margin: 0 auto`. max-width - will not allow the width to go beyond some value.

Reply:
This is great way to make the width responsive. Thanks man!